website/content/api-docs/secret/identity/tokens.mdx

```diff
@@ -421,9 +421,11 @@ $ curl \
 }
 ```
 
-## Read .well-known configurations
+## Read identity token issuer's OpenID configuration
```

Comment:
```suggestion
## Read the OpenID issuer configuration from an identity token
```
Style correction: avoid possessives

Reply:
The suggestion isn't technically correct since you can't read an OpenID issuer configuration from an identity token. The "identity token issuer" has an OpenID configuration. 

Happy to apply some other form of correction that you would advise 🙏

Reply:
Tweaked the suggestion

Reply:
Thank you!